crates/pallet-feeds/src/lib.rs

```diff
@@ -367,6 +368,18 @@ pub struct CallObject {
     pub offset: u32,
 }
 
+impl<T: Config> Pallet<T> {
+    pub fn successful_calls() -> Vec<T::Hash> {
+        SuccessfulCalls::<T>::get()
+    }
+}
+
+sp_api::decl_runtime_apis! {
```

Comment:
1) We only declare runtime APIs in the `sp-*` crates, let's not put them into the pallet
2) API should be named more descriptive like, `successful_calls` is too generic
3) The API itself shouldn't be tied to feeds, it should be generic, such that when we have another pallet that provides object mappings, it can be added to the runtime internals without changing public API (hence runtime API shouldn't be declared in `sp-feeds`)